### DISCUSSION TITLE
does deepgram supports IPA international phonetic alphabetical order ?

### DISCUSSION THREAD

**Discussion by Ashutosh-rawat-vassarlabs** [deepgram/community#994] [General help] [answered]
In TTS, when pronouncing Indian names, some names are pronounced correctly, but most are inaccurately rendered due to phonetic limitations. Is there an alternative solution available for accurately handling regional name pronunciation ?

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @Ashutosh-rawat-vassarlabs, our TTS pronunciation is not customizable at this time. You may be able to experiment with modifying spellings or helping the TTS "sound out" pronunciations, but otherwise there are not other modifications you can make. Here is our [doc on pronunciation](https://developers.deepgram.com/docs/text-to-speech-prompting#pronunciation).
